server/bootstrap/routes.js

```diff
@@ -5,16 +5,18 @@ var IndexController = require(process.env.ROOT + '/server/mvc/controllers/IndexC
 var AdminController = require(process.env.ROOT + '/server/mvc/controllers/AdminController.js');
 var Admin_UsersController = require(process.env.ROOT + '/server/mvc/controllers/Admin/UsersController.js');
 var ProfileController = require(process.env.ROOT + '/server/mvc/controllers/ProfileController.js');
+var BattlenetController = require(process.env.ROOT + '/server/mvc/controllers/BattlenetController.js');
 
 module.exports = function (app, logs, passport) {
   app.get('/auth/bnet', passport.authenticate('bnet'));
   app.get('/auth/bnet/callback', passport.authenticate('bnet', {failureRedirect: '/'}), function(req, res){
-    res.redirect('/');
+    res.redirect('/bnet');
```

Comment:
Maybe the BattlenetController could be used here instead of redirecting to the page where it'll be used. Am I missing something?